client/src/api/histories.ts

```diff
@@ -7,3 +7,7 @@ export const deleteHistories = fetcher.path("/api/histories/batch/delete").metho
 export const undeleteHistory = fetcher.path("/api/histories/deleted/{history_id}/undelete").method("post").create();
 export const undeleteHistories = fetcher.path("/api/histories/batch/undelete").method("put").create();
 export const historiesQuery = fetcher.path("/api/histories/query").method("get").create();
+
+export function getPublishedHistories() {
+    return historiesFetcher({ view: "details", q: ["published"], qv: ["True"] });
+}
```

Comment:
```suggestion
export const publishedHistoriesFetcher = fetcher.path("/api/histories/published").method("get").create();
```
There is a route for published histories that can be also used with the fetcher. I think this should take care of the selenium failure.